MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -394,6 +408,10 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
 					_texCoordTL, 
 					_texCoordBR);			
 			
+		    unchecked
+		    {
+                ++_drawCount;
```

Comment:
In this situation, it will make no difference in execution flow or performance either way.  In some situations in C++ it can be slightly better performance to have pre-increment (post-increment needs to make a copy of the value before incrementing and returning the copy), but for types such as integer it doesn't really matter.